hack/bin/deps.bzl

```diff
@@ -456,28 +455,43 @@ def install_preflight():
 def install_crdb():
     http_archive(
        name = "crdb_darwin", # todo fix or remove
-       sha256 = "bbbd0a75f81d3df4acd139fdc7f0961480161454db24f25263c9276c3959db54",
-       urls = ["https://binaries.cockroachdb.com/cockroach-v21.2.0.darwin-10.9-amd64.tgz"],
+       sha256 = "79fb1669678b802ae891ec3e005efa801c0f970fec4eb6af7ac89cdb6b991b42",
+       urls = ["https://binaries.cockroachdb.com/cockroach-v22.2.19.darwin-10.9-amd64.tgz"],
```

Comment:
Maybe use a newer version? This version is already not supported. Maybe use 24.3.1?

Reply:
Reason for being not use the latest version is the cockroach start command used in the statefulset has some arguments which are deprecated. Those arguments needs to be upgraded with backward compatibility to upgrade to latest version.  It lies outside the scope of this ticket

This is the reason I have upgraded to the minimum version where arm64 is supported.